sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1519,12 +1519,7 @@ class Analyzer(override val catalogManager: CatalogManager)
         }
       // If the script transformation input contains Stars, expand it.
```

Comment:
Please update the comment here (describe why it's okay just to pass `t.child.output` into `ScriptTransformation.input`).

Reply:
Done